api/v1alpha1/virtualmachineservice_types.go

```diff
@@ -93,6 +93,30 @@ type VirtualMachineServiceSpec struct {
 	// +optional
 	LoadBalancerIP string `json:"loadBalancerIP,omitempty"`
 
+	// LoadBalancerSourceRanges is an array of IP addresses in the format of
+	// CIDRs.
+	// For example, the following Service spec will allowlist IPs in
+	// 103.21.244.0/22 and 103.22.200.0/22.
+	//   apiVersion: v1
+	//   kind: Service
+	//   metadata:
+	//     name: myapp
+	//   spec:
+	//     ports:
+	//     - port: 443
+	//       targetPort: 9376
+	//     selector:
+	//       app: example
+	//     type: LoadBalancer
+	//     loadBalancerSourceRanges:
+	//     - 103.21.244.0/22
+	//     - 103.22.200.0/22
```

Comment:
Thanks for expanding the comment thoroughly :) but I don't think this level of detailed example is appropriate for a single field. I think simply stating something like "... in the format of CIDRs, for example 103.21.244.0/22 and 1.2.3.4" would be sufficient.

Reply:
makes sense. updated.